units/patch_intent.go

```diff
@@ -1083,8 +1097,17 @@ func (j *patchIntentProcessor) buildTriggerPatchDoc(patchDoc *patch.Patch) (*mod
 		}
 		for _, p := range parentPatch.Patches {
 			if p.ModuleName == "" {
+				moduleName := intent.ParentAsModule
```

Comment:
Is this special case behavior for child patches that have the parent as a module something that's documented/worth documenting?

Reply:
Sounds like a good idea. I added a sentence at the end of the patch trigger aliases docs mentioning this behavior.